.circleci/config.yml

```diff
@@ -724,19 +724,19 @@ workflows:
               only: production
       - approve-push-unstable:
           type: approval
-          filters:
-            branches:
-              only: main
+          # filters:
+          #   branches:
+          #     only: main
```

Comment:
We should remember to remove these before merge

Reply:
Done here: [350c959](https://github.com/shuttle-hq/shuttle/pull/1253/commits/350c959250e1e95014ab217f79ea457b82fc0442)